packages/cfd/src/Containers/cfd-password-manager-modal.tsx

```diff
@@ -74,7 +75,7 @@ const CFDPasswordReset = ({
 
     React.useEffect(() => {
         localStorage.setItem('cfd_reset_password_intent', [server, account_group, account_type].join('.'));
-        localStorage.setItem('cfd_reset_password_type', password_type);
```

Comment:
Do you know what's the value of `cfd_reset_password_type` ? I just want to know what we are storing in this field to make sure if it's a sensitive data or not. 
because if not, IMO it's too much to add a dependency to the project just for one property